apps/earn-protocol/app/server-handlers/sdk/get-user-positions.ts

```
@@ -1,37 +1,45 @@
+          return await backendSDK.armada.users.getUserPositions({
+            user,
+          })
+        }),
+      )
 
-    const positionsList = userPositionsAllNetworksCalls
-      .filter(Boolean)
-      .reduce((acc, position) => [...acc, ...position], [])
+      const positionsList = userPositionsAllNetworksCalls
+        .filter(Boolean)
+        .reduce((acc, position) => [...acc, ...position], [])
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)